### PR TITLE
fix issue 11450  Color contrast ratio of focus indicator over 'toolStripSplitButton1' is 1.247:1 which is less than 3:1: A11y_.NET CoreWinforms_StripControls_Toolstripwithmenuitem_NonTextContrast

### LINKED ISSUE #11450: Color contrast ratio of focus indicator over 'toolStripSplitButton1' is 1.247:1 which is less than 3:1: A11y_.NET CoreWinforms_StripControls_Toolstripwithmenuitem_NonTextContrast
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

_No response_

### Issue description

GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG1.4.11; #DesktopApp; #Win11; #FTP; #NonTextContrast;

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "MenuToolbars_Controls:Testing the controls under Menu Toolbars Tab" and hit ENTER key.
4. TAB to StripControls button and hit ENTER key.
5. TAB to 'toolStripSplitButton1' present at below section of the screen.
6. Observe the issue.

### Actual Result:
Color contrast ratio of focus indicator over 'toolStripSplitButton1' is 1.247:1 which is less than 3:1.

**Similar issue is also observed in below scenario:
Issue 1: ToolStripContainer**

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "MenuToolbars_Controls:Testing the controls under Menu Toolbars Tab" and hit ENTER key.
4. TAB to ToolStripContainer button and hit ENTER key.
5. TAB to 'toolStripSplitButton2' present at below section of the screen.
6. Observe that color contrast ratio over "toolStripSplitButton2" is failing.

**Issue2: PrintingTesting_PrintPreview:**

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Printing_Controls:Testing the controls under Printing Tab" and hit ENTER key.
4. TAB to Preview button and hit ENTER key.
5. Print Preview Screen would appear.
6. Navigate to "Zoom" button using TAB key and observe that color contrast ratio over "Zoom" is failing.

### Expected Result:
Color contrast ratio of focus indicator over 'toolStripSplitButton1' should be greater than or equal to 3:1

### User Impact:
Low vision users will not be able to identify where the focus is and will not be able to interact with the control.

### Attachment
11450_A11y_ NET CoreWinforms_StripControls_Toolstripwithmenuitem_NonTextContrast

### PR DESCRIPTION
**Pull request by Epica3055**
Fixes #11450
 
 
## Proposed changes
 
- 
- Remove RenderMode
-
 

 
 
### Before
 
image
 
### After
 
image
 

 
 
## Test environment(s)
 
 9.0.0-preview.6.24305.2
 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/11502)

### CODE REVIEW

**Comment by Tanya-Solyanik** [comment]
@Epica3055  - had you investigated if other items have the same issue? Olina's team might be able to help out

**Comment by Olina-Zhang** [comment]
@Tanya-Solyanik @Epica3055 I tested this PR, found an issue: Now the drop-down triangle display for toolStripSplitButton and toolStripDropDownButton looks are same in **StatusStrip**. But actually in **ToolStrip**, they are same before.

**Before:** the drop-down triangle of toolStripSplitButton is bigger than toolStripDropDownButton's. The toolStripSplitButton drop-down triangle size in StatusStrip and ToolStrip keep consistent now
image

**After:** the drop-down triangle size for toolStripSplitButton and toolStripDropDownButton looks are same in StatusStrip. The toolStripSplitButton in StatusStrip and ToolStrip keep consistent now
image

Don't know why the toolstripSplitButton's dropdown triangle is displayed differently in StatusStrip.

**Comment by ricardobossan** [comment]
The failing tests have to be addressed.

**Review by LeafShi1 — COMMENTED** [review]
LGTM! The description need to be updated.

File: src/System.Windows.Forms/src/System/Windows/Forms/Controls/ToolStrips/StatusStrip.cs [hunk 1]
```diff
@@ -428,17 +427,6 @@ protected override void SetDisplayedItems()
         base.SetDisplayedItems();
     }
 
-    internal override void ResetRenderMode()
-    {
-        RenderMode = ToolStripRenderMode.System;
-    }
-
-    internal override bool ShouldSerializeRenderMode()
```
Comment (lonitra) [inline src/System.Windows.Forms/src/System/Windows/Forms/Controls/ToolStrips/StatusStrip.cs:436]: Why did this need to be removed?

Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/src/System/Windows/Forms/Controls/ToolStrips/StatusStrip.cs:436]: This method is required for designer support - right click ->reset on the RenderMode property in the propertybrowser. Please test that the design-time scenario still works with your fix.

After this fix we are going with the ToolStrip value for RenderMode, so the ToolStrip implementations of Reset and ShouldSerialize  should be the right ones for StatusStrip

**Review by Tanya-Solyanik — APPROVED** [review]
Look good, but please double-check that design time serialization and reset in the property browser still work

**Comment by Epica3055** [comment]
@Tanya-Solyanik 

winforms_PR_11502

**Review by lonitra — APPROVED** [review]

**Pull request merged**